tests/integration/components/pipeline/workflow/component-test.js

```diff
@@ -36,6 +36,9 @@ module('Integration | Component | pipeline/workflow', function (hooks) {
 
   test('it renders for pipeline with no events', async function (assert) {
     sinon.stub(pipelinePageState, 'getIsPr').returns(false);
+    sinon.stub(shuttle, 'fetchFromApi')
+      .withArgs('get', `/pipelines/${pipelineId}/lastSuccessfulEvent`)
+      .throws(new Error('Not found test'));
```

Comment:
```suggestion
    sinon
      .stub(shuttle, 'fetchFromApi')
      .withArgs('get', `/pipelines/${pipelineId}/lastSuccessfulEvent`)
      .rejects();
```
The promise should reject instead of throwing an error